tools/cldr-apps/js/src/views/LookupPath.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <form action="">
```

Comment:
this html is derived from browser, running on production server

main change is "v-on:change"

formatted with npx prettier (default options)